frontend/public/ui/ui-actions.js

```diff
@@ -160,6 +164,17 @@ export const UIActions = {
 
   [types.setCreateProjectMessage]: message => ({type: types.setCreateProjectMessage, message}),
 
+  [types.setTableObject]: (crdModel) => dispatch => {
```

Comment:
This really feels like it belongs in k8s-actions / k8s-reducers. Shouldn't we generate a unique ID like `watchK8sList` etc?

Reply:
So originally I've used the k8s-actions / k8s-reducers, but after refactoring I've put it in UI, since it's keeping data that are being rendered. 
On the other hand putting it in k8s make sense as well.

Reply:
I think it makes more sense under k8s. It's data coming directly from the API and that's where other watch/list data is kept.